sponsor-dapp-v2/src/lib/routes.js

```diff
@@ -13,7 +13,7 @@ const routes = [
   { path: "/Start", component: StartScreen, exact: false },
   { path: "/Steps", component: Steps, exact: false },
   { path: "/ViewPositions", component: ViewPositions, exact: false },
-  { path: "/ManagePositions", component: ManagePositions, exact: false },
+  { path: "/ManagePositions/:tokenAddress", component: ManagePositions, exact: false },
```

Comment:
Just for my information, what does the colon do here?

Reply:
It matches whatever is in the URL after `/ManagePositions/` and provides it as a variable called `tokenAddress`.

Reply:
So cool!